### DISCUSSION TITLE
Welcome to dicom2stl Discussions!

### DISCUSSION THREAD

**Discussion by dave3d** [dave3d/dicom2stl#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jipiness** [comment]
Hi @dave3d, I was wondering if your dicom2stl is compatible with converting a dicom file that is already segmented?

**Comment by KryanWang** [comment]
Hi @dave3d,I didn't know anything about those code at all, but I'm very interested in your code. May I get some help to use those code from you?
